griptape/engines/rag/modules/query/related_query_generation_module.py

```diff
@@ -0,0 +1,33 @@
+from typing import Callable
+from attrs import define, field, Factory
+from griptape import utils
+from griptape.drivers import BasePromptDriver
+from griptape.engines.rag import RagContext
+from griptape.engines.rag.modules import BaseQueryModule
+from griptape.utils import J2
+
+
+@define(kw_only=True)
+class RelatedQueryGenerationModule(BaseQueryModule):
+    query_count: int = field(default=5)
+    prompt_driver: BasePromptDriver = field()
+    generate_system_template: Callable[[str], str] = field(
+        default=Factory(lambda self: self.default_system_template_generator, takes_self=True), kw_only=True
+    )
```

Comment:
Same as above, use of "generate"

Reply:
It's consistent with PromptTask.generate_system_template.